Formula/influxdb.rb

```diff
@@ -21,25 +21,45 @@ class Influxdb < Formula
 
   depends_on "bazaar" => :build
   depends_on "go" => :build
+  depends_on "make" => :build
   depends_on "pkg-config" => :build
+  depends_on "protobuf" => :build
   depends_on "rust" => :build
-  depends_on "yarn" => :build
-  depends_on "protobuf"
 
+  # NOTE: The version here is specified in the go.mod of influxdb.
+  # If you're upgrading to a newer influxdb version, check to see if this needs upgraded too.
   resource "pkg-config-wrapper" do
-    url "https://github.com/influxdata/pkg-config/archive/refs/tags/0.2.1.tar.gz"
-    sha256 "a31955bae060799b482d36ed522e76d55e1002d879d38371ed43d254b51f59d5"
+    url "https://github.com/influxdata/pkg-config/archive/refs/tags/v0.2.7.tar.gz"
+    sha256 "9bfe2c06b09fe7f3274f4ff8da1d87c9102640285bb38dad9a8c26dd5b9fe5af"
+  end
+
+  # NOTE: The version/URL here is specified in scripts/fetch-ui-assets.sh in influxdb.
+  # If you're upgrading to a newer influxdb version, check to see if this needs upgraded too.
+  resource "ui-assets" do
+    url "https://github.com/influxdata/ui/releases/download/OSS-v2.0.5/build.tar.gz"
+    sha256 "37ffbc072ba801ec5a0abdd76a3f19a8cd75f59856274e20630929f73cedaf55"
   end
 
   def install
+    # Set up the influxdata pkg-config wrapper to enable just-in-time compilation & linking
+    # of the Rust components in the server.
     resource("pkg-config-wrapper").stage do
       system "go", "build", *std_go_args, "-o", buildpath/"bootstrap/pkg-config"
     end
     ENV.prepend_path "PATH", buildpath/"bootstrap"
 
+    # Extract pre-build UI resources to the location expected by go-bindata.
+    resource("ui-assets").stage(buildpath/"ui/build")
+
+    ENV["GO111MODULE"] = "on"
     ldflags = "-s -w -X main.version=#{version}"
+
+    # Embed UI files into the Go source code.
+    system "make", "generate"
+
+    # Build the CLI and server.
     system "go", "build", *std_go_args, "-ldflags", ldflags, "-o", bin/"influx", "./cmd/influx"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags), "-o", bin/"influx", "./cmd/influx"
```
We've changed the syntax of `std_go_args` slightly, so let's update this since we're already making changes here.